sycl/source/detail/device_info.hpp

```diff
@@ -2223,6 +2223,167 @@ inline sycl::device get_device_info_host<
                       PI_ERROR_INVALID_DEVICE);
 }
 
+template <>
```

Comment:
same for device info host. aspect::host is deprecated https://github.com/intel/llvm/blob/a43ff24492fdb9ced7f2c906a593f20904d2a261/sycl/include/sycl/info/aspects_deprecated.def#L5
will it not compile now if we just not add get_device_info_host?

Reply:
Yeah, unfortunately I was getting compilation errors when I had not defined get_device_info_host.

Reply:
I see, ok then.